src/Types.tsx

```diff
@@ -0,0 +1,39 @@
+export type ResponseSectionsResults = {
```

Comment:
Also `.tsx` extension indicates that this is Typescript + JSX. But some of the files you've created do not contain any JSX inside so basically those can use `.ts`. This is minor issue (or even not issue at all) - just so you're aware of the differences between those two extensions.